bin/rename-project

```diff
@@ -51,6 +51,38 @@ find . -type f -exec \
     -pe "s/${FIND_MODULE_NAME}/${MODULE_NAME}/g;" {} +
 # -----------------------------------------------------------------------------
 
+cat << 'EOF'
```

Comment:
I think all of this can go away and you can drop a 1 liner to delete the contents of the directory in this section of the script `The core of the script which renames a few things.` and then adjust this comment to be something like `The core of the script.`

Given this rename script is likely used shortly after creating the project and temporary files are temporary, there's nothing of value to protect or warn against.